src/Components/Atoms/FATextField/index.tsx

```diff
@@ -1,6 +1,17 @@
 import React, { forwardRef, memo } from "react";
 import { TextField, InputAdornment } from "@mui/material";
 import { IInput } from "./Input.interface";
+import { makeStyles } from "@mui/styles";
+
+const useStyles = makeStyles(() => ({
```

Comment:
perlu konfirmasi ulang ke @muhammad-farhan64 untuk FATextField

Reply:
@muhammad-farhan64 tolong di respond ya thank you.